getting-started-2.0.md

```diff
@@ -0,0 +1,155 @@
+
+_Helix Pages_ can render a header and footer based on Markdown content in your GitHub repository:
+
+1. Add a file called `header.md` to your GitHub repository.
+   <br>![](./dummy.png "TODO: Create a header.md")
+1. Add a logo, and maybe a list of other links you would like to show in the header navigation. Here's some [inspiration](https://raw.githubusercontent.com/adobe/helix-example/master/header.md "example header.md").
+1. Click the _Commit changes_ button.
+1. If you added a logo in step 2, upload that now.
+   <br>![](./dummy.png "TODO: Upload logo")
+1. Add a file called `footer.md` to your GitHub repository.
+1. Add some content you would like to be shown in the footer. Here's a [proposal](https://raw.githubusercontent.com/adobe/helix-example/master/footer.md "example footer.md").
+1. Click the _Commit changes_ button.
+1. Reload `https://<my-repo>-<my_org>.project-helix.page/` to see header and footer in action.
+   <br>![TODO: Helix Pages rendering your header and footer](./dummy.png)
+
+## Step 3: Add your own HTML files
+
+> You'll need:
+> - basic HTML knowledge
+
+If you have existing HTML files or snippets that you wish to reuse, you can simply add them to your GitHub repository and have _Helix_Pages_ merge and deliver them for you:
+
+1. Add a `static.html` file to your GitHub repository. The file name can be anything, just make sure it has a `.html` extension.
+1. Add your HTML. Maybe [like so](https://raw.githubusercontent.com/adobe/helix-example/master/htdocs/static.html "example HTML file")?
+1. Click _Commit changes_.
+1. Refresh `https://<my-repo>-<my_org>.project-helix.page/static.html` to see the rendered result.
+
+> **Note:** If there is a `.md` and `.html` with the same name, _Helix Pages_ will give precendence to the Markdown file. Rename or remove the Markdown file to have the HTML file rendered instead.
+
+> **Hint:** If you add a static `header.html` or `footer.html` instead of their Markdown versions, _Helix Pages_ will embed these for you.
+
+## Add your own styling
+
+> You'll need:
+> - basic CSS knowledge
+
+_Helix Pages_ allows you to override the default CSS. You can either do this directly in GitHub, or start developing locally.
+
+### GitHub
+
+1. Add a `style.css` file to your GitHub repository.
+1. Insert your CSS rules. See the [default CSS](https://raw.githubusercontent.com/adobe/helix-pages/master/htdocs/style.css "default CSS") for pointers where you can add overrides.
+1. Click _Commit changes_.
+1. Reload `https://<my-repo>-<my_org>.project-helix.page/` to view your style changes in action.
+1. Repeat steps 2 - 4 until you are happy with how your page looks.
+
+### Local development
+
+For an improved developer experience, using a local environment and the IDE of your choice, we proudly introduce you to the _Helix CLI_ (`hlx`). For the sake of simplicity, we just assume you are using _Visual Studio Code_ and have a `code` alias for it.
+
+> You'll need:
+> - a shell :)
```

Comment:
what is a shell? :shell: ?

- a command line terminal (linux, macos) or command prompt (windows)

but since every OS has some sort of terminal/command prompt, it is not really a requirement.

Reply:
Here I thought of the thing running _inside_ the terminal and executing your commands, i.e. `bash` or `zsh` or... but you're right, it doesn't matter which you use, and every OS provides one. The key is to find the terminal ;)